dist/origin-web-common-services.js

```diff
@@ -3193,6 +3194,9 @@ angular.module('openshiftCommonServices')
                                         });
                               }, function(e) {
                                 context.projectPromise.reject(e);
+                                if ((e.status === 403 || e.status === 404) && _.has(opts, 'skipErrorOnProjNotFound')) {
+                                  return $q.reject({projNotFound: true});
```

Comment:
Let's go with either `notFound: true` or `projectNotFound: true`. (I have a slight preference for the first.)

Reply:
Fixed to 'notFound: true'